src/pages/player/player.jsx

```diff
@@ -1,19 +1,19 @@
 var React = require('react');
-var {parseJSON} = require('../../../node_modules/webmaker-core/src/lib/jsonUtils');
-var Loading = require('../../../node_modules/webmaker-core/src/components/loading/loading.jsx');
+var {parseJSON} = require('webmaker-core/src/lib/jsonUtils');
+var Loading = require('webmaker-core/src/components/loading/loading.jsx');
 var {Menu, PrimaryButton, FullWidthButton} = require('../../../node_modules/webmaker-core/src/components/action-menu/action-menu.jsx');
-var PageBlock = require("../../../node_modules/webmaker-core/src/pages/project/pageblock.jsx");
+var PageBlock = require("webmaker-core/src/pages/project/pageblock.jsx");
 
 module.exports = React.createClass({
   mixins: [
-    require('../../../node_modules/webmaker-core/src/lib/router'),
-    require('../../../node_modules/webmaker-core/src/pages/project/transforms'),
-    require('../../../node_modules/webmaker-core/src/pages/project/remix'),
-    require('../../../node_modules/webmaker-core/src/pages/project/cartzoom'),
-    require('../../../node_modules/webmaker-core/src/pages/project/pageadmin'),
-    require('../../../node_modules/webmaker-core/src/pages/project/loader'),
-    require('../../../node_modules/webmaker-core/src/pages/project/setdestination'),
-    require('../../../node_modules/webmaker-core/src/pages/project/renderhelpers')
+    require('webmaker-core/src/lib/router'),
```

Comment:
Whenever you need to access files inside a node module, you can do so by having them relative to the module name instead of the actual path